accessibility/accessibility-in-emails.md

```diff
@@ -0,0 +1,63 @@
+# Accessibility in emails
+
+This info sits alongside our general guide to writing emails at ????.
+
+## Intro
+
+Accessibility in emails can be summarised with 3 key points:
+
+1. Accessible emails allow you to reach a wider audience.
+1. Writing and visual design are key components of an accessible email campaign.
+1. Email templates can be made immediately more accessible with a few simple coding techniques.
+
+Creating accessible emails are not fundamentally different from creating accessible websites.
+
+Here are the few differing constraints and techniques that should be considered.
+
+## Layout
+
+- Rich visual emails typically use tables for layout.
+- This is because email clients use old and conservative browser engines that do not cope well nor consistently with CSS-based layouts.
+- Using ARIA to set `role="presentation"` on layout tables will help prevent them being seen as data tables by screen readers.
+
+## Specify a language
+
+- Setting the language of the email on the root element will allow assistive technology to use the appropriate language profile.
+
+## Alternative text
+
+- Marketing emails tend to make heavy use of images.
+- Alt text should be used as it would be on a web site
+- If the image conveys meaning, add an alt description.
+- If the image is purely presentational, add an empty alt attribute (`alt=""`) to allow the screenreader to skip it.
+- If the image is an image of text, the alt attribute must contain that same text.
+- Never leave an image without an alt attribute as the assistive tech will read out the filename instead. If you're sure the image doesn't need alt text, use an empty alt attribute (`alt=""`). 
+- Do not use alt text to prompt the user to enable images in their email client.
+
+## Use logical source order
```

Comment:
Maybe move the logical source order section up a bit so it goes directly below the "specify a language" section?